### ISSUE TITLE
Docker-Image 3.0.0 Buggy on apple chips

### ISSUE THREAD

**Issue by hdruse** [tomasbjerre/violations-command-line#10]
Running violations-command-line in docker sometimes result in following error:

```
The current machine does not support all of the following CPU features that are required by the image: [CX8, CMOV, FXSR, MMX, SSE, SSE2, SSE3, SSSE3, SSE4_1, SSE4_2, POPCNT, LZCNT, AVX, AVX2, BMI1, BMI2, FMA].
Please rebuild the executable with an appropriate setting of the -march option.
```

The error is not happening all the time. and i never encountered it with previous versions though the script for creating the docker image did not change.

**Comment by tomasbjerre** [comment]
Are you using the image from Dockerhub?

"the script for creating the docker image " do you mean you are extracting the binary from the image? How?

The big thing in 3.0.0 was using native compiled binary in the image, not a jar. The jar is available in Maven Central.

**Comment by hdruse** [comment]
I am using the Image from Dockerhub.

> "the script for creating the docker image " do you mean you are extracting the binary from the image? How?

I am refering to [build-docker.sh](https://github.com/tomasbjerre/violations-command-line/blob/master/build-docker.sh)

Running on a mbpro with m3 pro

![image](https://github.com/tomasbjerre/violations-command-line/assets/88705919/577dd4a0-d845-49f2-ba0d-9f196b4928a8)

**Comment by tomasbjerre** [comment]
I published `3.0.2` now that reverts back from native to a runnable jar.

Thanks for reporting, open issue again if not working.
